exercises/armstrong-numbers/example.scm

```diff
@@ -0,0 +1,17 @@
+(import (rnrs))
+
+(define (list-digits n)
+  (let loop ([n n]
+             [ms '()])
+    (let-values ([(d m) (div-and-mod n 10)])
+      (if (zero? n)
```

Comment:
Not hugely important, but it probably makes sense to check `zero?` before calling `div-and-mod`